<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>990</width>
    <height>725</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>990</width>
    <height>725</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>990</width>
    <height>725</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Corte</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #222433;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="QT_Imagem">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>50</y>
      <width>768</width>
      <height>576</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #222433;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>660</y>
      <width>951</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="QT_btReferencia">
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color: rgb(23, 25, 33);
color: #E62E2E;
}
QPushButton:hover{
	background-color: #1B1D26;
}</string>
       </property>
       <property name="text">
        <string>Referência</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; color:#e62e2e;&quot;&gt;Posição&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="QT_posicao">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(23, 25, 33);
color: #E62E2E;
</string>
       </property>
       <item>
        <property name="text">
         <string>Superior Esquerdo</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Superior Direito</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Inferior Esquerdo</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Inferior Direito</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>200</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="QT_btFinalizar">
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color: rgb(23, 25, 33);
color: #E62E2E;
}
QPushButton:hover{
	background-color: #1B1D26;
}</string>
       </property>
       <property name="text">
        <string>Salvar</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
